src/app/modules/core/components/burger-menu/burger-menu.component.html

```diff
@@ -0,0 +1,114 @@
+      fill="#0E0C08"/>
+    <path
+      d="M14.306 16.965c.384.383 1.15-.384 1.917-1.15.767-.767 1.917-1.534 1.534-2.3-.384-.768-2.3-.384-2.684-.768-.383-.383-.383-.767-1.15-.767-.532.024-1.053.154-1.534.384-.686.686-1.533 1.15.384 1.534 1.917.383 2.684.766 2.3 1.917-.084.252-1.15.766-.767 1.15z"
+      fill="#0E0C08"/>
+    <path
+      d="M13.54 16.201c.383 0 .766-.383 1.15-.766.383-.384-.767-.384-1.15-.384-.384 0-1.04-.264-1.15 0-.208.502.766 1.15 1.15 1.15z"
+      fill="#0E0C08"/>
+  </symbol>
+
+  <symbol fill="none" id="arrow-down" viewBox="0 0 24 24">
+    <path clip-rule="evenodd" d="M6 9.293l6 6 6-6 .707.707L12 16.707 5.293 10 6 9.293z" fill="#0E0C08"
+          fill-rule="evenodd"/>
+  </symbol>
+</svg>
+
+<div (click)="closeBurgerMenu()" class="page-container">
+  <div (click)="$event.stopPropagation()" class="burger-wrapper">
```

Comment:
replace with window listener - https://stackoverflow.com/questions/51150422/how-to-detect-click-outside-of-an-element-in-angular

Reply:
Fixed